### DISCUSSION TITLE
HELLO, What is this!?

### DISCUSSION THREAD

**Discussion by aliyabatoolalvi** [community/community#68098] [GitHub Education]
### Hi everyone,

I am new here and exploring git hub student developer pack, if you have any suggestions , do share with me.

**Comment by yalza** [comment]
Response #2

**Comment by mantasu** [comment]
Depends on your career path, but I would highly recommend using the learning resources, for example you have 6 months free for [Educative](https://www.educative.io/github-students) and 3 months premium plan for [DataCamp](https://www.datacamp.com/pricing/student). After finishing some courses you can put the certificate on your LinkedIn.

I also just re-applied for GitHub for students solely for [Copilot](https://github.com/features/copilot) (which is free for student), and specifically to try its voice feature. If you plan to write some code, try enrolling to [Copilot Voice](https://githubnext.com/projects/copilot-voice/) waitlist and after getting access you may be able to write code or notes very efficiently using a [voice assistant extension](https://marketplace.visualstudio.com/items?itemName=GitHub.heygithub).

**Reply by aliyabatoolalvi** [reply]
Thank you. @mantasu 
I am planning to adopt copilot as well for my development projects. But I just knew about [Copilot Voice](https://githubnext.com/projects/copilot-voice/), its innovative.
